VAMobile/src/screens/HealthScreen/Appointments/UpcomingAppointments/AppointmentCancellationInfo.tsx

```diff
@@ -44,7 +44,13 @@
   let body
   let bodyA11yLabel
 
-  if (isCovidVaccine) {
+  if (phoneOnly) {
+    title = t('upcomingAppointmentDetails.doYouNeedToCancelOrReschedule')
+    body =
+      (appointmentType === AppointmentTypeConstants.VA || phoneOnly) && !isCovidVaccine && cancelId
```

Comment:
fixed

Reply:
appointmentType === AppointmentTypeConstants.VA

This code is irrelevant as written, since phoneOnly will always be true?

Reply:
phoneOnly is not always true so need to check if it is either or

Reply:
phoneOnly is always true here because it's inside

```
if (phoneOnly) {
  ...
}
```

If you hover over phoneOnly, you can see the TS compiler has detected that, because it shows `const phoneOnly: true` instead of `const phoneOnly: boolean`

Reply:
ah, good catch

updated